jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/RadioButtonPage.java

```diff
@@ -0,0 +1,35 @@
+package io.github.com.pages.inputs;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+import java.util.List;
+
+public class RadioButtonPage extends WebPage {
+    @UI("//h2[text()='Simple radio']/following-sibling::fieldset[1]/div/label")
+    public static List<UIElement> simpleRadioButtons;
+
+    @UI("//h2[text()='Simple radio']/following-sibling::fieldset[1]/div/label/span[1]")
+    public static List<UIElement> simpleRadioButtonsClass;
+
+    @UI("//h2[text()='Simple radio']/following-sibling::fieldset[1]/p")
```

Comment:
please use this locator
//p[@id='lastClickContent']
same for others

Reply:
done